### PR TITLE
Exchanges: Add config variable to set bypassing of orderbook verification by exchange

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

Allow user to set orderbook verification bypass by the configuration file for an individual exchange. This assumes the user has deemed the exchange to have correct bid-ask alignment and stable data. This affects all exchanges and all protocols. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Travis with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: config/config_types.go [hunk 1]
```diff
@@ -131,6 +131,7 @@ type ExchangeConfig struct {
 	API                             APIConfig              `json:"api"`
 	Features                        *FeaturesConfig        `json:"features"`
 	BankAccounts                    []banking.Account      `json:"bankAccounts,omitempty"`
+	OrderbookVerificationBypass     bool                   `json:"orderbookVerificationBypass"`
```
Comment (gloriousCode) [inline config/config_types.go:134]: We could potentially group these now

```
type OrderbookConfig struct {
	VerificationBypass bool                              `json:"orderbookVerificationBypass"`
	WebsocketBufferLimit   int                         `json:"websocketOrderbookBufferLimit"`
	WebsocketBufferEnabled bool                   `json:"websocketOrderbookBufferEnabled"`
}
```

**Review by gloriousCode — APPROVED** [review]
tACK! It works, as risky as it is 😄

File: exchanges/binance/binance_wrapper.go [hunk 2]
```diff
@@ -401,7 +401,12 @@ func (b *Binance) FetchOrderbook(p currency.Pair, assetType asset.Item) (*orderb
 
 // UpdateOrderbook updates and returns the orderbook for a currency pair
 func (b *Binance) UpdateOrderbook(p currency.Pair, assetType asset.Item) (*orderbook.Base, error) {
-	book := &orderbook.Base{ExchangeName: b.Name, Pair: p, AssetType: assetType}
+	book := &orderbook.Base{
+		ExchangeName:       b.Name,
+		Pair:               p,
+		AssetType:          assetType,
+		VerificationBypass: b.OrderbookVerificationBypass,
+	}
```
Comment (gloriousCode) [inline exchanges/binance/binance_wrapper.go:409]: This is good for REST, however, for the websocket, the `b.OrderbookVerificationBypass` isn't being set in `SeedLocalCacheWithBook`/`SeedLocalCache`.

It's also happening for ZB I've noticed, so please double check websocket implementations. 


I'm just logging every time validation is run for an exchange, so if I see it when I set bypass to true, I know something is up.

**Review by gloriousCode — CHANGES_REQUESTED** [review]
Thanks for making that change! It made me test things a little more and have discovered an issue

File: exchanges/binance/binance_wrapper.go (hunk 2 above)
Comment (shazbert) [reply exchanges/binance/binance_wrapper.go:409]: Will check it out!

**Review by MadCozBadd — APPROVED** [review]
tACK!!!!!

**Review by gloriousCode — APPROVED** [review]
Thanks for fixing that last bit! tACK

**Review by thrasher- — APPROVED** [review]

**Comment by thrasher-** [comment]
FYI race spotted but as discussed this can be fixed in a separate PR: https://ci.appveyor.com/project/thrasher-/gocryptotrader-g5pk0/builds/37126977#L676

**Pull request merged**